pkg/registry/scheduling/priorityclass/storage/storage.go

```diff
@@ -56,3 +61,14 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"pc"}
 }
+
+// Delete ensures that system priority classes are not deleted.
```

Comment:
Should we have guard for update as well or else we will able to update system priority class to lower value and then create pods that have higher priority than system priority?

Reply:
Update validation already prevents changing value

Reply:
Ohh I missed that, it seems it is part of https://github.com/kubernetes/kubernetes/pull/60519/files#diff-35601e9483eab62205294e45c6df7712R71. Thanks.